docker/dockerfile/Dockerfile

```
@@ -16,7 +15,14 @@ RUN mkdir -p /etc/apt/keyrings \
     && echo "deb [signed-by=/etc/apt/keyrings/postgresql.gpg] http://apt.postgresql.org/pub/repos/apt bookworm-pgdg main" \
     > /etc/apt/sources.list.d/pgdg.list
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN curl -fsSL https://nginx.org/keys/nginx_signing.key -o /tmp/nginx_signing.key && \
+    gpg --dearmor -o /etc/apt/keyrings/nginx-archive-keyring.gpg /tmp/nginx_signing.key && \
+    rm /tmp/nginx_signing.key && \
+    echo "deb [signed-by=/etc/apt/keyrings/nginx-archive-keyring.gpg] http://nginx.org/packages/debian/ bookworm nginx" \
+    > /etc/apt/sources.list.d/nginx.list
+
+RUN apt-get clean && apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)